### ISSUE TITLE
Installation fails on Windows with the pre-generated command

### ISSUE THREAD

**Issue by dend** [pytorch/pytorch#48545]
## 🐛 Bug

When selecting the following combination (Stable + Windows + PIP + Python + CUDA 11.0), the generated command errors out on a Windows machine.

![image](https://user-images.githubusercontent.com/1389609/100527529-199d8900-3188-11eb-8728-b06652f68930.png)

Command in question:

```bash
pip install torch===1.7.0+cu110 torchvision===0.8.1+cu110 torchaudio===0.7.0 -f https://download.pytorch.org/whl/torch_stable.html
```

It seems like Python 3.9 is not supported?

## To Reproduce

1. Go to https://pytorch.org/get-started/locally/.
1. Set the selection to the one above.
1. Copy generated command.
1. Attempt to use the generated command on Windows.

![image](https://user-images.githubusercontent.com/1389609/100527557-6aad7d00-3188-11eb-8869-312f97e3f8be.png)

## Expected behavior

PyTorch is installed on Windows.

## Environment

 - PyTorch Version (e.g., 1.0): Not installed.
 - OS (e.g., Linux): Windows 10 20H2
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source): see above.
 - Python version: 3.9.0
 - CUDA/cuDNN version: 11.1
 - GPU models and configuration: RTX 2080 Super KO
 - Any other relevant information: N/A

## Additional context

cc @ezyang @seemethere @malfet @walterddr

**Comment by peterjc123** [comment]
Python 3.9 is unsupported yet. And it will be fixed by the upcoming 1.7.1 release.

**Comment by ejguan** [comment]
BTW, should we mention supported python version in https://pytorch.org/get-started/locally/
